packages/vm/tests/api/index.spec.ts

```diff
@@ -11,7 +11,6 @@ import * as testData from './testdata.json'
 
 // explicitly import util and buffer,
 // needed for karma-typescript bundling
-import * as util from 'util'
```

Comment:
Is removing this ok despite the comment above?

Reply:
Didn't see this comment, will re-enable it with an eslint-disable just to be sure that we don't get problems for browser bundling.

Reply:
Will keep it in with an eslint-disable, d873cd9